destreamer.ts

```diff
@@ -99,11 +99,7 @@ async function DoInteractiveLogin(username?: string): Promise<Session> {
         await page.click('input[type="submit"]');
     }
 
-    // Going trough "remember me" pagin (choosing NO)
-    await page.waitForSelector('input[type="button"]', {timeout: 120000});
-    await page.click('input[type="button"]');
-
-    await browser.waitForTarget(target => target.url() == "https://web.microsoftstream.com/")
+    await browser.waitForTarget(target => target.url() == "https://web.microsoftstream.com/", {timeout: 150000});

```

Comment:
this change should be reverted too, as snobu said url is not equal to web.microsoftstream.com for all

Reply:
Yup. We should await a url that includes microsoftstream.com. It's a safe bet for now.

Reply:
Ok if you just want to check for includes "micorsosftstream.com" and not match the whole URL we should also che that it doesn't end with "nosigneuprequirer" because that page redirects to the final landing page and we cannot evaluate our session there because it doesn't exist yet

Reply:
Looks like you are absolutely spot on with that finding. Thanks!
Please send a PR against the tokencache branch for this fix, i'm losing track of what i did and didn't merge in from `dev`. That's nobody's fault but mine and my super poor planning.

Reply:
Maybe we can go one step further? Can we have puppeteer await the presence of the access token in the cookie and then proceed with session extraction?